packages/components/organisms/f-content-cards/src/components/_tests/ContentCards.test.js

```diff
@@ -66,39 +70,38 @@ const scopedSlots = {
 
 beforeEach(() => {
     jest.resetAllMocks();
-    initialiseMetadata.mockResolvedValue(metadataDispatcher);
 });
 
 describe('ContentCards', () => {
-    it('should call intitialiseMetadata when mounted', () => {
+    it('should call BrazeAdapter when mounted', () => {
         // Arrange & Act
-        shallowMount(ContentCards, {
+        const wrapper = shallowMount(ContentCards, {
             propsData: {
                 apiKey,
                 userId
             },
             scopedSlots: {
                 default: '<div></div>'
-            }
+            },
+            mocks
         });
 
         // Assert
-        const [[settings]] = initialiseMetadata.mock.calls;
-        expect(settings.apiKey).toBe(apiKey);
-        expect(settings.userId).toBe(userId);
-        expect(settings.enableLogging).toBe(false);
+        expect(wrapper.vm.brazeAdapter).toBeInstanceOf(BrazeAdapter);
     });
 
 
 
-    describe('when a rejection from the promise returned by intitialiseMetadata has been encountered during loading', () => {
+    describe('when a error is thrown from the instantiation of BrazeAdapter during loading', () => {
```

Comment:
Nit, but feel free to fix later

```suggestion
    describe('when an error is thrown from the instantiation of BrazeAdapter during loading', () => {
```